client/package.json

```diff
@@ -27,11 +27,13 @@
 		"clsx": "^1.1.1",
 		"formik": "^2.2.9",
 		"jwt-decode": "^3.1.2",
+		"moment": "^2.29.1",
```

Comment:
moment is a very heavy library
it is usually recommended to use something lighter, like dayjs (which is super light) or similar
but i think for our project moment is fine